core/src/main/java/com/linecorp/armeria/server/file/MimeTypeUtil.java

```diff
@@ -70,6 +70,18 @@
         EXTENSION_TO_MEDIA_TYPE = Collections.unmodifiableMap(map);
     }
 
+    private static final MediaTypeResolver DEFAULT_MEDIA_TYPE_RESOLVER = new MediaTypeResolver() {
```

Comment:
How about refactoring `MimeTypeUtil` to extend `MediaTypeResolver`?
```java
enum DefaultMediaTypeResolver implements MediaTypeResolver {
    INSTANCE;
    ...

    @Override
    @Nullable
    public MediaType guessFromPath(String path) {
         ...
    }
    ...
}

public interface MediaTypeResolver {
    static MediaTypeResolver ofDefault() {
        return DefaultMediaTypeResolver.INSTANCE;
    }
}
```

Reply:
Could you also migrate the code where `MimeTypeUtil.guessFromPath()` is directly used to `MediaTypeResolver.ofDefault().guessFromPath()`?

Reply:
> How about refactoring MimeTypeUtil to extend MediaTypeResolver?

You suggested me to refactor `MimeTypeUtil(MediaTypeUtil)` to enum `DefaultMediaTypeResolver` that implement `MediaTypeResolver `instead, right? Please correct me if I'm wrong, Thanks.

> Could you also migrate the code...

Noted. I'll do that.

Reply:
> You suggested me to refactor MimeTypeUtil(MediaTypeUtil) to enum DefaultMediaTypeResolver that implement MediaTypeResolver instead, right? Please correct me if I'm wrong, Thanks.

It is right.